flex/pkg/volume/driver-call.go

```diff
@@ -30,7 +30,7 @@ const (
 	provisionCmd = "provision"
 	deleteCmd    = "delete"
 
-	optionPVorVolumeName = "kubernetes.io/pvOrVolumeName"
+	optionPVorVolumeName = "pvOrVolumeName"
```

Comment:
Just for future reference we can put one comment abt removing "kubernetes.io" also doing continue while getFlexVolumeOptions()